src/services/UserService.js

```diff
@@ -41,6 +41,7 @@ function makeAjaxRequest(method, path, body, callback) {
 
   request.open(method.toUpperCase(), path, true);
   request.setRequestHeader('Accept', 'application/json');
+  request.setRequestHeader('X-Stormpath-Agent', pkg.name + '/' + pkg.version);
```

Comment:
Please see my comment on @robertjd's [PR](https://github.com/stormpath/stormpath-sdk-angularjs/pull/107) regarding if we should include the framework version as well (in this case `react/x.x.x`).